buildpack/telemetry/appdynamics.py

```diff
@@ -51,8 +67,31 @@ def update_config(m2ee, app_name):
         )
 
 
+def run():
+    if not (appdynamics_used() and machine_agent_enabled()):
+        return
+
+    logging.info("Starting the appDynamics Machine agent...")
+    e = dict(os.environ)
+    subprocess.Popen(
+        (AGENT_PATH, "-Dmetric.http.listener=true"),
+        env=e,
+    )
+
+
 def appdynamics_used():
     for key, _ in os.environ.items():
         if key.startswith("APPDYNAMICS_"):
             return True
     return False
+
+
+def machine_agent_enabled():
+
+    machine_agent_enabled = bool(
+        int(os.getenv("APPDYNAMICS_MACHINE_AGENT_ENABLED", default=0))
```

Comment:
From looking at the other envvars, it's usually the string `True` or `False`. So let's be consistent and have the var as a string.